src/Status/Views.elm

```diff
@@ -0,0 +1,39 @@
+module Status.Views exposing (view)
+
+import Html exposing (Html, text, button, div, li, ul, b)
+import Html.Events exposing (onClick)
+import Status.Types exposing (StatusMsg(GetStatus), StatusResponse)
+import Types exposing (Msg(MsgForStatus), Model)
+
+
+view : Model -> Html Types.Msg
+view model =
+    div []
+        [ button [ onClick (MsgForStatus <| GetStatus) ] [ text "Get Status" ]
```

Comment:
We'll talk with @w0rm about it tomorrow, but he sent me this message:
```
I kinda like the translator pattern more. It avoids requiring the main modules
from the submodules.

Why should the status module know about the message type of the parent app?

With this approach it will be easier to run into circular dependencies

Plus the translator pattern is only needed when we need to communicate from
child to parent. It seems that it is not needed for the Status module
```

So, we'll have a chat and see what happens!